portfolio/src/main/webapp/script.js

```diff
@@ -2,42 +2,64 @@
 
 let elementsToShow;
 let scroll;
+let quotes =
+    ["\"In a world too often governed by corruption and arrogance, it can be difficult to stay true to one's philosophical and literary principles.\"   -Lemony Snicket, A Series of Unfortunate Events", "\"Did you know that the Sparrow flies south for winter?\"   -Skulduggery Pleasant, Skulduggery Pleasant", "\"Someday in your life, you'll realize there is no turning back. There was never any turning back.\"   -u/JayStar1213"];
+let entrances = 
+    ["animate__bounceInUp", "animate__bounceInLeft", "animate__bounceInRight", "animate__bounceInDown", "animate__fadeInBottomLeft", "animate__fadeInBottomRight"];
 
 $(document).ready(function() {
+    // scroll here is a fallback function in case some browswers don't support method requestAnimationFrame. 
     scroll = window.requestAnimationFrame || function(callback){ window.setTimeout(callback, 1000/60)};
-    elementsToShow = document.querySelectorAll('.animate');    
+    // all the elements that require css animation. 
+    elementsToShow = document.querySelectorAll('.animate');  
+
     loop();
+
     asyncRemovePlaceHolder();
 })
 
-function addRandomGreeting() {
+/**
+ * Get a random quote from the quotes array and gets a random css animation class from the entrances array for the quote to be displayed
+ */
+function getRandomQuote() {
 
+    // see function doc on line 57
     revertToUnchecked();
 
-    const greetings =
-        ["\"In a world too often governed by corruption and arrogance, it can be difficult to stay true to one's philosophical and literary principles.\"   -Lemony Snicket, A Series of Unfortunate Events", "\"Did you know that the Sparrow flies south for winter?\"   -Skulduggery Pleasant, Skulduggery Pleasant", "\"Someday in your life, you'll realize there is no turning back. There was never any turning back.\"   -u/JayStar1213"];
     const greeting = greetings[Math.floor(Math.random() * greetings.length)];
   
-    let entrances = 
-        ["animate__bounceInUp", "animate__bounceInLeft", "animate__bounceInRight", "animate__bounceInDown", "animate__fadeInBottomLeft", "animate__fadeInBottomRight"]
     let entrance = entrances[Math.floor(Math.random() * greetings.length)];
 
-    const quote = document.getElementById('quote');
+    
+    let quote = document.getElementById('quote');
```

Comment:
seems like `entrance` and `quote` are only set once, so use `const` instead

Reply:
changed to const